pkgs/applications/networking/n8n/default.nix

```
@@ -38,12 +38,6 @@ nodePackages.n8n.override {
   meta = with lib; {
     description = "Free and source-available fair-code licensed workflow automation tool. Easily automate tasks across different services.";
     maintainers = with maintainers; [ freezeboy k900 ];
-    license = {
-      fullName = "Sustainable Use License";
-      url = "https://github.com/n8n-io/n8n/blob/master/LICENSE.md";
-      free = false;
-      # only free to redistribute "for non-commercial purposes"
-      redistributable = false;
-    };
+    license = lib.licences.sutainable;
```

```suggestion
    license = licences.sutainableUse;
```

The license name is wrong, and the `lib.` is unnecessary because you're inside a `with`.

---

Apologies, an American/British English difference. Fixed.

---

It's not fixed.  Did you try evaluating n8n?  The license name still doesn't match what you put in licenses.nix.

---

I was using the `shortName` incorrectly, apologies for the back-and-forth. It builds now.

```shell
$ pwd; nix-build -A n8n
/data/nixpkgs
/nix/store/a5ggq3ydqg7l8x6aw46qgqgx6wmzvbwx-n8n-0.227.1
```